storage/imx6ull-flash/flashdev.c

```diff
@@ -99,7 +247,7 @@ static int flashmtd_erase(struct _storage_t *strg, off_t offs, size_t size)
 static int flashmtd_read(struct _storage_t *strg, off_t offs, void *data, size_t len, size_t *retlen)
 {
 	uint32_t paddr;
-	int err = EOK;
+	int err, ret = EOK;
```

Comment:
Do we need two variables for handling return value? The question applies for the other changes.

Reply:
Yes, we do need 2 variables because of `-EUCLEAN` error status which allows to continue normal operation. One variable is needed to know whether `-EUCLEAN` happened and the second one for current error checks.